code/game/objects/items/granters/martial_arts/_martial_arts.dm

```diff
@@ -16,9 +16,11 @@
 
 /obj/item/book/granter/martial/on_reading_start(mob/user)
 	to_chat(user, span_notice("You start reading about [martial_name]..."))
+	if(SSticker.current_state == GAME_STATE_FINISHED)//faster after the round ends
+		pages_to_mastery = 0
```

Comment:
You can probably just call ``on_reading_finished`` and early return

Reply:
i just did that in a different proc so i wouldn't need to modify other subtypes